resolwe_bio/tests/workflows/test_amplicon.py

```diff
@@ -58,6 +61,11 @@ def test_amplicon_workflow(self):
                 },
                 'var_annot': {
                     'known_vars_db': [dbsnp.id]
-                }
+                },
+                "report": {
```

Comment:
Use single quotes here and in the two lines below to be consistent with the rest of the file.